tests/Stepper.test.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { Stepper } from "components";
+
+const STEPS = [
+  { id: "1", label: "Connect your calendar" },
+  { id: "2", label: "Connect conferencing app" },
+  { id: "3", label: "Set availability" },
+];
+
+const getActiveClass = () =>
+  document.querySelector(".neeto-ui-stepper-item__wrapper--active");
+
+const getDoneClass = () =>
+  document.querySelector(".neeto-ui-stepper-item__wrapper--done");
+
+const ControlledStepper = () => {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  return (
+    <Stepper
+      activeIndex={activeIndex}
+      setActiveIndex={setActiveIndex}
+      steps={STEPS}
+    />
+  );
+};
+
+describe("Stepper", () => {
+  it("should render without error", () => {
+    const { getByText } = render(<ControlledStepper />);
+    STEPS.forEach(({ id, label }) => {
+      expect(getByText(id)).toBeInTheDocument();
+      expect(getByText(label)).toBeInTheDocument();
+    });
```

Comment:
Use `screen`, update other instances as well

```suggestion
    render(<ControlledStepper />);
    STEPS.forEach(({ id, label }) => {
      expect(screen.getByText(id)).toBeInTheDocument();
      expect(screen.getByText(label)).toBeInTheDocument();
    });
```